pubspec.yaml

```diff
@@ -27,6 +30,14 @@ flutter:
         pluginClass: DesktopWebviewAuthPlugin
       windows:
         pluginClass: DesktopWebviewAuthPlugin
+      ios:
+        pluginClass: DesktopWebviewAuthPlugin
+      android:
+        package: com.example.desktop_webview_auth
```

Comment:
Not sure about the package name here, why is it `example`?

Reply:
it comes from default `flutter create` template. doesn't really matter because this won't be used, just needs to be here for pub.dev to be able to figure out that forebase_ui_oauth supports mobile and web as well.